packages/core/src/modules/basic-messages/messages/BasicMessage.ts

```diff
@@ -1,11 +1,11 @@
 import { Expose, Transform } from 'class-transformer'
 import { IsDate, IsString } from 'class-validator'
 
-import { AgentMessage } from '../../../agent/AgentMessage'
+import { DidCommV1Message } from '../../../didcomm'
 import { IsValidMessageType, parseMessageType } from '../../../utils/messageType'
 import { DateParser } from '../../../utils/transformers'
 
-export class BasicMessage extends AgentMessage {
+export class BasicMessage extends DidCommV1Message {
```

Comment:
Current implementation of basic-messages is V1 so it's obviously a DidCommV1Message but I'm wondering what we are going to do in protocols that are specified for both DidComm V1 and V2. This is the case of Basic Message 2.0 (https://didcomm.org/basicmessage/2.0/) and Shorten URL 1.0 (https://didcomm.org/shorten-url/1.0/).